### DISCUSSION TITLE
GITHUB USERNAME FOR MY PROFILE TO BE SAVED

### DISCUSSION THREAD

**Discussion by MB006358** [community/community#39346] [New to GitHub]
I AMSTUCK WITH TRYING TO CREATE A GITHUB USERNAME FOR MY PROFILE TO COMPLETE MY ON BOARDING
I HAVE DONE ALL THAT CAN. IT IS LEFT FOR ME TO GIVE UP UNLESS I GET HELP

**Comment by subidit** [comment]
Have you changed your username before?
Please be aware that you could change your username only once, so proceed carefully. 
Anyway to change username, from
main menu > settings > account > change username. 

It's the first option in account page. 

Hope it helps.

**Comment by jfullstackdev** [comment]
you will not be able to post here if you did not succeed creating your username, you also have your display name

![image](https://user-images.githubusercontent.com/47092464/202896492-9e3f79b6-2fe9-411e-be1c-6eb6726cc5fc.png)
